modules/javafx.controls/src/main/java/javafx/scene/control/TextFormatter.java

```diff
@@ -37,7 +37,7 @@
  * A Formatter describes a format of a {@code TextInputControl} text by using two distinct mechanisms:
  * <ul>
  *     <li>A filter ({@link #getFilter()}) that can intercept and modify user input. This helps to keep the text
- *     in the desired format. A default text supplier can be used to provide the intial text.</li>
+ *     in the desired format. A default text supplier can be used to provide the initial text.</li>
```

Comment:
I know this is a simple typo, but it is unrelated to your bug fix, and is in public API docs, so I'd like to see it go in separately under a "Fix mistakes in docs" bug. I filed [JDK-8286678](https://bugs.openjdk.java.net/browse/JDK-8286678) to track this and any other such issues that arise (as we've done for most recent releases).

Reply:
Ah okay, alright. I reverted it.